topostats/filters.py

```diff
@@ -424,14 +424,14 @@ def filter_image(self) -> None:
         run_scar_removal = self.remove_scars_config.pop("run")
         if run_scar_removal:
             LOGGER.info(f"[{self.filename}] : Initial scar removal")
-            self.images["initial_scar_removal"], _scar_mask = scars.remove_scars(
+            self.images["initial_scar_removal"], _ = scars.remove_scars(
```

Comment:
One for subsequent work but if the scar mask is not required we could remove it from being returned (#714).